app/views/home/index.html.haml

```diff
@@ -1,3 +1,7 @@
-%h1 Home
-= link_to "Training Modules", training_modules_path
-%p Find me in app/views/home/index.html.haml
+.govuk-grid-row
+  .govuk-grid-column-two-thirds
+    %h1.govuk-heading-xl
+      Home
+    %p.govuk-body 
+      Your application is ready
+    = configuration_summary_list
```

Comment:
It would be nice if this page could still include the link to the training modules as that's useful for testing login

Reply:
I put it in the navigation, which seems like a better place (until we decide where it actually goes)

Reply:
Oh! I missed that - you're right that is a better place.